src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServicesCmdletBase.cs

```diff
@@ -0,0 +1,261 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using Microsoft.Azure.Commands.MachineLearning.Extensions;
+using Microsoft.Azure.Commands.MachineLearning.Utilities;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Entities.ErrorResponses;
+using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Rest.Azure;
+using PSResourceManagerModels = Microsoft.Azure.Commands.Resources.Models;
+
+namespace Microsoft.Azure.Commands.MachineLearning
+{
+    public abstract class WebServicesCmdletBase : AzureRMCmdlet
+    {
+        public const string CommandletSuffix = "AzureRmMlWebService";
```

Comment:
Note that you can cosider adding shorter cmdlet aliases for this release or in a subsequent release.